Src/FluentAssertions/Primitives/ReferenceTypeAssertions.cs

```diff
@@ -15,10 +15,15 @@ namespace FluentAssertions.Primitives
     public abstract class ReferenceTypeAssertions<TSubject, TAssertions>
         where TAssertions : ReferenceTypeAssertions<TSubject, TAssertions>
     {
+        protected ReferenceTypeAssertions(TSubject subject)
```

Comment:
Thanks for raising this point about it being a breaking change for consumers. To be honest it didn't occur to me.

I'll go with your suggestion as it's a good way of solving the problem. I think it might also be wise if I add a unit test to verify that ReferenceTypeAssertions definitely has a parameterless constructor, given its importance.

Reply:
It's on my wishlist to introduce API Approval tests 
https://github.com/JakeGinnivan/ApiApprover

Reply:
Resolved this issue and added that unit test

Reply:
Could you do the same for the three other classes mentioned above?

Reply:
Done :) let me know if I missed anything